README.md

```diff
@@ -102,6 +102,11 @@ Or, for `jest`:
 const {jest: jestConfig} = require('kcd-scripts/config')
 module.exports = Object.assign(jestConfig, {
   // your overrides here
+
+  // for test written in Typescript, add:
```

Comment:
Since `kcd-scripts` won't just work out of the box for Typescript tests because I felt the transform was too specific to put into kcd-script; I thought it would be a nice thing to document what was needed to do so.  What do you think?

Reply:
Looks good :+1: